sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -16,6 +16,16 @@ select interval '2 seconds' / 0;
 select interval '2 seconds' / null;
 select interval '2 seconds' * null;
 select null * interval '2 seconds';
+select interval '2 year' / 0;
+select interval '2 year' / null;
+select interval '2 year' * null;
+select null * interval '2 year';
```

Comment:
Can you check interval in ANSI style at least once: `interval '2' year`

Reply:
Done, updated.